.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release Docker Image
+
+on:
+  push:
+    tags:
+      - "*"
+    branches:
+      - "*"
```

Comment:
I think we should only release new docker containers on new tag.

Reply:
+1